src/tags/domain/tag.ts

```diff
@@ -0,0 +1,18 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class Tag {
+  @ApiProperty({
+    type: String,
+  })
+  id: string;
+
+  @ApiProperty()
+  name: string;
+
+  // @custom-inject-point
+  @ApiProperty()
+  created_at: Date;
```

Comment:
Initially, we used camelCase for database names and entity column names, but we later switched to snake_case. Let me explain why we made that change.

1. PostgreSQL explicitly recommends [not using uppercase](https://wiki.postgresql.org/wiki/Don't_Do_This#Don.27t_use_upper_case_table_or_column_names) in column names.
2. While your suggestion isn't about changing the names in the database but rather just in the entity names, I generally believe that persistence entities should closely match the database structure. If any conversion is required, we can always handle that in the domain layer. We have mappers in place where we can change the domain column names without altering the entity or database column names.
3. So yes, I agree that TypeScript and NestJS generally work with `camelCase`, but since PostgreSQL recommends against it, we decided to follow the database recommendations and keep the persistence entities and columns in sync.